packages/tizen-remote/lib/index.js

```diff
@@ -626,6 +792,8 @@ export class TizenRemote extends createdTypedEmitterClass() {
       this.#debug(`Requesting new token; waiting ${this.#tokenTimeout / 1000}s...`);
       this.#token = await this.getToken();
       this.#debug('Received token');
+    } else if (this.#token) {
+      await this.writeToken(this.#token);
```

Comment:
we don't want to check whether the user wants to persist the token before writing?

Reply:
I think you're right